admin/app/routes/authenticated/organizations/get.js

```diff
@@ -3,6 +3,11 @@ import { service } from '@ember/service';
 
 export default class GetRoute extends Route {
   @service store;
+  @service oidcIdentityProviders;
```

Comment:
@clemlatz quid de le faire sinon sur la route authenticated ?

https://github.com/1024pix/pix/blob/073c0390beebf70a2f8414aaaa0f889287df3954/admin/app/routes/authenticated.js#L1-L12

le seul + que je vois à faire ça c'est que si un agent pix passe d'une page utilisateur à une organisation, il ne rappelera pas la route

Reply:
mais j'ai l'impression de décaler le problème avec cette proposition

Reply:
Oui ça me paraît une bonne idée. Cet appel là doit être global (car il n'est utilisé dans plusieurs pages de Pix Admin) mais survenir uniquement quand l'utilisateur est authentifié.

L'autre appel qui ne retournera que les "ready for pix admin" pourra lui ne se faire que dans la route login parce qu'on en n'a besoin qu'à cet endroit.

On pourrait déjà prévoir d'avoir des méthodes de service avec des noms distincts comme par exemple `loadAllAvailableIdentityProviders` et `loadIdentityProvidersReadyForPixAdmin`.